backend/Functions/Edna.AssignmentLearnContent/Edna.AssignmentLearnContent/Startup.cs

```diff
@@ -23,7 +24,8 @@ public void Configure(IWebJobsBuilder builder)
 
             builder.AddUserBinding();
 
-            builder.Services.AddHttpClient();
+            builder.Services.AddHttpClient(EdnaExternalHttpHandler.Name).ConfigurePrimaryHttpMessageHandler(() => new EdnaExternalHttpHandler());

```

Comment:
Is it possible to have an `EdnaHttpClient` which does this automatically? That way, we could simply put the following in all the files allowing us to manage the configurations and code hardening at a single place.
``` C#
builder.Services.AddEdnaHttpClient();

// Implementation could be something like below
public AddEdnaHttpClient(this Services services) {
  services
    .AddHttpClient(EdnaExternalHttpHandler.Name)
    .ConfigurePrimaryHttpMessageHandler(() => new EdnaExternalHttpHandler());
}
```

Reply:
That would be useful! Let me try and get that working, I will push the changes shortly!